frontend/app/services/work-package-service.js

```diff
@@ -25,6 +25,7 @@
 //
 // See doc/COPYRIGHT.rdoc for more details.
 //++
+/* globals URI */
```

Comment:
What?

This line was added during a commit called "address hound complaints".

Either the complaint was

> Please add a useless comment mentioning URIs

or it is some kind of unwanted commit.